src/functions/txc-analysis-reporter/index.ts

```diff
@@ -311,11 +328,26 @@ export const handler: Handler = async (event, context) => {
                 archive.append(observationByObservationTypeCsv, {
                     name: `${date}/advisoryObservationsByObservationType/${observationType}.csv`,
                 });
+
+                if (STAGE === "prod") {
+                    dqsS3Promises.push(
+                        putS3Object({
+                            Bucket: DQS_BUCKET_NAME,
+                            Key: `${date}/advisoryObservationsByObservationType/${observationType}.csv`,
+                            ContentType: "application/csv",
+                            Body: observationByObservationTypeCsv,
+                        }),
+                    );
+                }
             }
         }
 
         archive.finalize();
         await s3Upload.done();
+
+        if (dqsS3Promises.length > 0) {
+            await Promise.allSettled(dqsS3Promises);
```

Comment:
Actually maybe do just make this a Promise.all, we probably do want the lambda to error if it can't write to the dqs bucket